### DISCUSSION TITLE
Osu keeps on freezing then mess up the audio and timing

### DISCUSSION THREAD

**Discussion by Bloqqy** [ppy/osu#29573] [Q&A]
### Type

Performance

### Bug description

Everytime me osu freezes, It will mess up the timing with the audio and offset also, this was a problem for me in multi since i had no idea why i was doing bad and kept on hitting 100's and 50's

In the video I uploaded, I was already doing badly because it had freezed, but after the 2nd one i couldn't find the rhythm/beat and failed.

### Screenshots or videos

https://github.com/user-attachments/assets/d75439ad-94e0-47a3-ad8f-41d98c4e7323

### Version

2024.817.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16719285/compressed-logs.zip)

**Comment by bdach** [comment]
Please upload video of this with frame graph shown (press ctrl-f11 twice).
